BennyTordrupVisma/Week1/Ass1/index.html

```diff
@@ -0,0 +1,88 @@
+<html>
+<head>
+	<title>Hello world without react</title>
+	<style type="text/css">
+	body {
+		width: 1000px;
+		margin: auto;
+	}
+	.error-message {
+		color: red;
+	}
+	</style>
+</head>
+<body>
+
+	<!-- 1. Create an inputfield that only accepts a string -->
+	<!-- 2. Find a way to submit and print the message -->
+	<!-- 3. Every message should be printed inside the message-container -->
+	<!-- 4. Validation errors in inputfields should produce an error-message rather than a hello morld message -->
+	<!-- 5. Any error-message should be printed in red color -->
+	<!-- 6. Provide a function that returns all messages ever printed -->
+	<!-- 7. This must be solved by using only native Javascript -->
+	<!-- 8. All messages must be printed in the order they where submitted -->
+
+	<div id="message-container"></div>
+	<input id="input" type="text">
+	<input id="commit" type="button" value="Click" onclick="onClick();"/>
+	<input id="showAll" type="button" value="View all" onclick="displayAllMessages();"/>
```

Comment:
Any reason for you not using `<button />` instead of setting att. type on inputs?
Also, there's no reason to set the value or id on the buttons as you don't use them. It just more code to read.